tools/docker/start.sh

```diff
@@ -1,14 +1,24 @@
 #!/bin/bash
+set -ex
 
-cd web-platform-tests
-git pull --depth=50
+REPO=${1:-origin}
+BRANCH=${2:-master}
+REV=${3:-master}
+BROWSER=${4:-all}
+
+cd ~/web-platform-tests
+git fetch ${REPO} ${BRANCH} -n -q --depth=50
+git checkout -b build ${REV}
```

Comment:
Why create a build branch?

Reply:
Why not? It avoids a useless warning about checking out a refless revision, avoids having to suppress that warning, and potentially makes things a little easier to work with for interactive debugging.

Reply:
OK, so there's a warning to suppress :) If not for that, it'd just be more code.